services/clojars/clojars-base.js

```diff
@@ -0,0 +1,65 @@
+'use strict'
+
+const Joi = require('joi')
+const { version: versionColor } = require('../color-formatters')
+const { nonNegativeInteger } = require('../validators')
+const { BaseJsonService, InvalidResponse } = require('..')
+
+const clojarsSchema = Joi.object({
+  downloads: nonNegativeInteger,
+  latest_version: Joi.string().required(),
+  latest_release: Joi.string().required(),
+}).required()
+
+class BaseClojarsService extends BaseJsonService {
+  async fetch({ clojar }) {
+    const url = `https://clojars.org/api/artifacts/${clojar}`
+    return this._requestJson({
+      url,
+      schema: clojarsSchema,
+      errorMessages: {
+        404: 'not found',
```

Comment:
`not found` is the default error message displayed automatically. You can use this if you want to customize that message (for example other services do things like `repo not found`, `project not found`, etc.), but if you're good with the default `not found` message then you don't need this

Reply:
removed `errorMessages`